hack/common.sh

```diff
@@ -73,4 +74,73 @@ function delete_cluster {
     config_dir="$(cluster_config_dir "${CLUSTER_NAME}")"
 
     openshift-install destroy cluster --dir="${config_dir}" --log-level=debug 2>&1 | tee -a "${log_file}"
-}
\ No newline at end of file
+}
+
+function jsonnet_build {
+    CONFIG="${1}"
+
+    rm -rf "${MANIFESTS_ROOT}"
+    mkdir -p "${MANIFESTS_ROOT}"
+    jsonnet "${JSONNET_ROOT}/main.jsonnet" > "${MANIFESTS_ROOT}/main.json" --tla-code config="${CONFIG}"
+}
+
+function run_benchmarks {
+    run_root="${1}"
+    PODS_PER_NODE="${2}"
+    POD_CHURNING_PERIOD="${3}"
+    NUMBER_OF_NS="${4}"
+
+    mkdir -p "${run_root}"
+
+    echo "Aborting previous uncompleted runs"
+    benchmarks_ns=$(kubectl get namespaces | grep -i prometheus-sizing | awk '{ print $1 }')
+    if [ -n "${benchmarks_ns}" ]
+    then
+        benchmarks_ns_arr=(${benchmarks_ns})
+        for ns in "${benchmarks_ns_arr[@]}"
+        do
+            kubectl delete namespace "${ns}"
+        done
+    fi
+    export PODS_PER_NODE=${PODS_PER_NODE}
```

Comment:
Do we really need to export these if they are already defined in the `env` vars of the `ReplicaSet`?

Reply:
That allows to run `make run/benchmarks` locally. But I agree it would be cleaner to assume those variables are also exported even locally, so I removed the export and adapted the doc comments to make clear that assumption